pytools/tag.py

```diff
@@ -94,6 +95,28 @@ def from_class(cls, argcls: Any) -> "DottedName":
 tag_dataclass = dataclass(init=True, eq=True, frozen=True, repr=True)
 
 
+@tag_dataclass
```

Comment:
```suggestion
```

Reply:
Or was there a specific reason? This isn't itself a tag, so `@tag_dataclass` doesn't seem to make sense.